frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -1307,6 +1307,35 @@ export function reserveDatasetUpload(
   );
 }
 
+export type OngoingUpload = {
+  uploadId: string;
+  dataSourceId: { name: string; organizationName: string };
```

Comment:
usually we call this datasetId in the frontend (see `APIDatasetId`). can we use that term here too?